### DISCUSSION TITLE
Manual data loading / filtering

### DISCUSSION THREAD

**Discussion by tyteen4a03** [komarovalexander/ka-table#252] [Q&A]
What is the preferred strategy for having ka-table be a dumb table row renderer, and letting an external data source (e.g. an API) handle row filtering?

I would like to rely 100% on the API to handle the filter operation but ka-table doesn't seem to have examples of these externally-controlled tables, and tries to do its own filtering on top of the filtered rows.

**Comment by komarovalexander** [comment]
Hi @tyteen4a03 
you can override filtering operation by this option
```
filter: () => () => true,
```
demo: https://stackblitz.com/edit/table-filter-row-ts-6tve7i?file=Demo.tsx
